Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -616,8 +616,11 @@ GstBuffer* gstBufferNewWrappedFast(void* data, size_t length)
 GstElement* makeGStreamerElement(const char* factoryName, const char* name)
 {
     auto* element = gst_element_factory_make(factoryName, name);
-    if (!element)
+    if (!element) {
         WTFLogAlways("GStreamer element %s not found. Please install it", factoryName);
+        GST_WARNING("GStreamer element %s not found. Please install it", factoryName);
```

Comment:
Why this new warning? `gst_element_factory_make()` already emits one when its return value is NULL :)